### PR TITLE
feat: Next 12 upgrade

### PR DESCRIPTION
**Pull request by ccorda**
This update Next 12, and all node deps. As a part of bumping deps, bumped Node to v16 (current LTS, a few packages complained about)

### CODE REVIEW

**Comment by ccorda** [comment]
Need to downgrade ES Lint, it'll be fixed in Next 12.0.5, but that's still in canary:
https://github.com/vercel/next.js/pull/29865

File: website/package.json [hunk 1]
```diff
@@ -20,39 +20,39 @@
...
+    "bootstrap": "^5.1.3",
     "classnames": "^2.3.1",
-    "dayjs": "^1.10.6",
+    "dayjs": "^1.10.7",
     "isomorphic-unfetch": "^3.1.0",
-    "next": "^11.1.0",
-    "next-seo": "^4.26.0",
+    "next": "^12.0.4",
+    "next-seo": "^4.28.1",
     "react": "^17.0.2",
```
Comment (kylehotchkiss) [inline website/package.json:30]: next, react, and react-dom should be updated in tandem. Are react/react-dom both on latest version?

Comment (ccorda) [reply website/package.json:31]: I use this to update deps: https://www.npmjs.com/package/npm-check-updates

It's the latest version of everything except for eslint which has issues (see comment earlier in thread). The current version of react is 17.0.2.

**Review by chrisherold — APPROVED** [review]
lgtm

**Pull request merged**